cmd/limactl/list.go

```diff
@@ -128,27 +141,31 @@ func listAction(cmd *cobra.Command, args []string) error {
 	}
 
 	instanceNames := []string{}
-	unmatchedInstances := false
+	unmatchedInstanceNames := []string{}
 	if len(args) > 0 {
 		for _, arg := range args {
 			matches := instanceMatches(arg, allinstances)
 			if len(matches) > 0 {
 				instanceNames = append(instanceNames, matches...)
 			} else {
-				logrus.Warnf("No instance matching %v found.", arg)
-				unmatchedInstances = true
+				unmatchedInstanceNames = append(unmatchedInstanceNames, arg)
```

Comment:
Why change?

Reply:
Reverted.

It's a refactoring that can be done in a separate PR.